clientComponents/src/main/java/ru/edu/spbstu/clientComponents/HboxCellWithCheckboxes.java

```diff
@@ -0,0 +1,61 @@
+package ru.edu.spbstu.clientComponents;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Priority;
+import ru.edu.spbstu.model.ChatUser;
+
+public class HboxCellWithCheckboxes extends HBox {
+
+    public static class MyCheckBox extends CheckBox {
+        protected int structNum;
+        MyCheckBox()
+        {
+            super();
+        }
+        MyCheckBox(int num)
+        {
+            super();
+            structNum=num;
+        }
+
+    }
+    Label label=new Label();
+    MyCheckBox deleteCB=new MyCheckBox();
+    MyCheckBox adminCB=new MyCheckBox();
```

Comment:
Похоже, эти поля создаются дважды.
```suggestion
    Label label;
    MyCheckBox deleteCB;
    MyCheckBox adminCB;
```